benchmark/initialize-benchmark-entities.sh

```diff
@@ -9,9 +9,10 @@ function usage {
   echo "Usage: $(basename $0) -k KEYCLOAK_HOME -r REALM_NAME -c CLIENT_ID [-d]" 2>&1
   echo '-k keycloak home path ex: /home/opt/keycloak-18.0.0 : default value KEYCLOAK_HOME env variable'
   echo '-d delete the client and realm before re-creating them: default value is false'
-  echo '-r realm : default value realm-0'
+  echo '-r realm : default value test-realm'
   echo '-c service account enabled client name : default value gatling'
   echo '-d delete the client and realm : default value is false'
+  echo '-u user : default username user-0 and default password is user-0-password'
```

Comment:
Reading the code it is currently not possible to override the password on the CLI. This behavior is ok, and the help should be aligned with this. See below for an attempt to do so:

 ```suggestion
   echo '-u user : default username user-0. The user\'s password will always be the username plus the suffix -password, for example, for user-0 this would be user-0-password'
```